src/main/java/org/reaktivity/nukleus/mqtt/internal/MqttController.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright 2016-2019 The Reaktivity Project
+ *
+ * The Reaktivity Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.reaktivity.nukleus.mqtt.internal;
+
+import static java.nio.ByteBuffer.allocateDirect;
+import static java.nio.ByteOrder.nativeOrder;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.agrona.MutableDirectBuffer;
+import org.agrona.concurrent.UnsafeBuffer;
+import org.reaktivity.nukleus.Controller;
+import org.reaktivity.nukleus.ControllerSpi;
+
+import org.reaktivity.nukleus.route.RouteKind;
+import org.reaktivity.nukleus.mqtt.internal.types.Flyweight;
+import org.reaktivity.nukleus.mqtt.internal.types.OctetsFW;
+import org.reaktivity.nukleus.mqtt.internal.types.control.Role;
+import org.reaktivity.nukleus.mqtt.internal.types.control.RouteFW;
+import org.reaktivity.nukleus.mqtt.internal.types.control.UnrouteFW;
+import org.reaktivity.nukleus.mqtt.internal.types.control.MqttRouteExFW;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+public final class MqttController implements Controller
+{
+    private static final int MAX_SEND_LENGTH = 1024;
+
+    private final RouteFW.Builder routeRW = new RouteFW.Builder();
+    private final UnrouteFW.Builder unrouteRW = new UnrouteFW.Builder();
+
+    private final MqttRouteExFW.Builder routeExRW = new MqttRouteExFW.Builder();
+
+    private final OctetsFW extensionRO = new OctetsFW().wrap(new UnsafeBuffer(new byte[0]), 0, 0);
+
+    private final ControllerSpi controllerSpi;
+    private final MutableDirectBuffer commandBuffer;
+    private final MutableDirectBuffer extensionBuffer;
+    private final Gson gson;
+
+    public MqttController(
+            ControllerSpi controllerSpi)
```

Comment:
Indentation for method parameters is doubled. Not sure why checkstyle is not catching this.
Please review and fix on all similar files.